src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -19,10 +20,10 @@
 public class PersonUtil {
 
     /**
-     * Returns an add command string for adding the {@code person}.
+     * Returns an add command string for adding the {@code participant}.
      */
-    public static String getAddCommand(Person person) {
-        return AddCommand.COMMAND_WORD + " " + getPersonDetails(person);
+    public static String getAddCommand(Participant participant) {
+        return AddCommand.COMMAND_WORD + " " + getPersonDetails(participant);
```

Comment:
Not sure how necessary these changes are since ParticipantUtil already exists

Reply:
I over-refactored thanks